rpm/gssdp.spec

```diff
@@ -1,23 +1,20 @@
 Name:          gssdp
-Version:       1.0.1
-Release:       1%{?dist}
+Version:       1.2.3
```

Comment:
Is it possible to update to the latest `1.3.0` release?

Reply:
Done. The old mirror hadn't picked up these new releases.

Reply:
Update also the PR title to match the new version.